src/components/AddressTable.tsx

```diff
@@ -110,16 +111,19 @@ export const AddressTable = ({
           title="Address"
           dataIndex="key"
           key="key"
-          render={(key) => (
-            <a
-              className="lattice-a"
-              href={`https://etherscan.io/address/${key}`}
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              {abbreviateHash(key)}
-            </a>
-          )}
+          render={(key) =>
+            isValidEthAddress(key)
+              ? (
+                <a
+                  className="lattice-a"
+                  href={`https://etherscan.io/address/${key}`}
```

Comment:
As mentioned in the issue, this will be confusing to users on e.g. Polygon, since those addresses look just like Ethereum addresses.

I think we can just remove the link all together for now, unless you have a better iday

Reply:
Outside of users labelling that data, I don't think there's a way to reliably detect the correct network for linking.

Reply:
[6503fb4](https://github.com/GridPlus/lattice-manager/pull/221/commits/6503fb452a2772002efcca700c4d50ae25f17238)